docs/src/pages/components/grid/InteractiveGrid.tsx

```diff
@@ -23,8 +23,10 @@ type GridJustification =
   | 'space-around'
   | 'space-evenly';
 
+type Direction = Exclude<GridProps['direction'], undefined>;
```

Comment:
This kind of prooves my point. Now you need to know `Exclude` and then create this helper type when you could just re-use the existing one.